backend/src/main/java/com/google/blackswan/mock/SimpleAnomalyGenerator.java

```diff
@@ -59,7 +59,14 @@ private SimpleAnomalyGenerator(InputStream source, int threshold,
     // Find instances where exceed threshold.
     Map<Timestamp, Integer> anomalyPoints = data.entrySet().stream()
         .filter(entry -> entry.getValue() - avg > threshold)
-        .collect(toMap(entry -> entry.getKey(), entry -> entry.getValue()));
+        .collect(toMap(
+            entry -> entry.getKey(), 
+            entry -> entry.getValue(),
+            // In case of conflicting keys, pick value of second key. 
+            // But there should not be conflicting keys at all. 
+            (x, y) -> y,
+            LinkedHashMap::new
+          ));
```

Comment:
to confirm: this is still mapping key to value and the exact map is a `LinkedHashMap`, but the `(x, y) -> y` means to pick the value of the second key?

Reply:
yep, that is correct